pkl-core/src/main/java/org/pkl/core/runtime/VmExceptionRenderer.java

```diff
@@ -59,7 +59,8 @@ private void renderBugException(VmBugException exception, StringBuilder builder)
         .append("Cmd+Double-click the link below to open an issue.\n")
         .append(
             "Please copy and paste the entire error output into the issue's description, provided you can share it.\n\n")
-        .append("https://github.com/apple/pkl/issues/new");
+        .append("https://github.com/apple/pkl/issues/new")
+        .append("\n\n");
```

Comment:
Yeah, except maybe keep it brief
```suggestion
        .append("https://github.com/apple/pkl/issues/new\n\n");
```

Reply:
@holzensp sure I have made the changes.